src/cli.rs

```diff
@@ -100,9 +100,9 @@ pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
             Arg::with_name("identity")
                 .short("i")
                 .long("identity")
-                .value_name("PATH")
+                .value_name("FILEPATH")
                 .takes_value(true)
-                .help("File containing a client identity (keypair)"),
+                .help("Identity used to connect with QUIC connection, identity with lot of stakes have better chances to send transaction to the leader"),
```

Comment:
I think it is better to rephrase it like this.
Also, you can wrap long string literals, to fit the recommended 100 characters maximum line width.
```suggestion
                .help(
                    "Identity used in the QUIC connection. Identity with a lot of stake has a \
                     better chance to send transaction to the leader"
                ),
```